tests/reg_tests/test_euler_tutorial_wing.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import print_function
+import unittest
+import numpy
+from baseclasses import BaseRegTest
+import os
+import sys
+baseDir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(baseDir,'../../'))
```

Comment:
One thing I just noticed since I'm working on pyHyp tests, is that this append should probably be an insertion, i.e. `sys.path.insert(0,...)`. The order in `sys.path` is important, and if  the parent folder is last on the path then another set of codes could be found first, meaning you are not testing the codes you think you are testing.